share/spice/chefio/chefio.js

```diff
@@ -0,0 +1,36 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_chefio = function(api_result){
+        if (!api_result || api_result.error) {
+            return Spice.failed('chefio');
+        }
+
+        Spice.add({
+            id: "chefio",
+            name: "software",
+            data: api_result.items,
+            meta: {
+                sourceName: "Chef.io",
+                sourceUrl: "http://chef.io"
+            },
+            normalize: function(item) {
+                var a = {
```

Comment:
Yeah true, should read my code better next time when using a template. Sorry about that!